client/components/ReviewRating/WriteReview/WriteReview.jsx

```diff
@@ -49,132 +60,215 @@ const WriteReview = ({
   const onSubmit = (e) => {
     setIsValid(true);
     e.preventDefault();
+    const errBlock = [];
     if (!newReview.summary) {
-      document.getElementById('summary').classList.add('error');
-      document.getElementById('summary').placeholder = 'can not be empty';
-      setIsValid(false);
+      document.getElementById('summary').placeholder = 'Example: Best purchase ever!';
+      errBlock.push('Summary');
     }
     if (!newReview.body) {
-      document.getElementById('body').classList.add('error');
-      document.getElementById('body').placeholder = 'can not be empty';
-      setIsValid(false);
+      errBlock.push('Body');
+      document.getElementById('body').placeholder = 'Why did you like the product or not?';
```

Comment:
I think you could be setting yourself for trouble by giving elements the same Id as <body>, especially when it's not the <body> element. Can you give it a more descriptive id like `formBody` or something like that.